installer/dev/install.cpp

```diff
@@ -289,6 +300,40 @@ namespace WindowsAppRuntimeInstaller
     }
     CATCH_RETURN()
 
+    // RestartPushNotificationsLRP is best effort and non-blocking to Installer functionality
+    void RestartPushNotificationsLRP()
+    {
+        WindowsAppRuntimeInstaller::InstallActivity::Context::Get().SetInstallStage(WindowsAppRuntimeInstaller::InstallActivity::InstallStage::RestartPushNotificationsLRP);
+
+        IID pushNotificationsIMPL_CLSID;
+        HRESULT hClsIdFromString = CLSIDFromString(PUSHNOTIFICATIONS_IMPL_CLSID_WSTRING, &pushNotificationsIMPL_CLSID);
+
+        if (SUCCEEDED(hClsIdFromString))
+        {
+            IID pushNotificationsLRP_IID;
+            hClsIdFromString = CLSIDFromString(PUSHNOTIFICATIONS_LRP_CLSID_WSTRING, &pushNotificationsLRP_IID);
+
+            if (SUCCEEDED(hClsIdFromString))
+            {
+                wil::com_ptr<::IUnknown> pNotificationsLRP{};
+                LOG_IF_FAILED(CoCreateInstance(
```

Comment:
Why not use the wil equivalent? WRL::CoCreateInstanc is old style right?

Reply:
Installer exe is built outside of the WAR dll build pipeline and hence, it doesn't have access to runtime generated header files. Hence, I had to switch to using compile time header and to using WRL::CoCreateInstance due to having no access to NotificationsLRP Interface header file at compile time. Or is there one that I can use ? I will be very happy, if that's the case.